app/uk/gov/hmrc/helptosavefrontend/views/iv/user_aborted.scala.html

```diff
@@ -16,26 +16,23 @@
 
 @import uk.gov.hmrc.helptosavefrontend.models.HtsContext
 @import uk.gov.hmrc.helptosavefrontend.config.FrontendAppConfig
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+@import uk.gov.hmrc.helptosavefrontend.views.ViewHelpers
 
-@this(main: uk.gov.hmrc.helptosavefrontend.views.html.main_template)
+@this(layout: uk.gov.hmrc.helptosavefrontend.views.html.layout, viewHelpers: ViewHelpers)
 
 @(retryUrl: String)(implicit htsContext: HtsContext, request: Request[_], messages: Messages, appConfig: FrontendAppConfig)
 
-@title = @{
-  Messages("hts.iv.were_unable_to_confirm_your_identity.title.h1")
-}
-
-@main(
-  title = title
-) {
-
-  <h1 class="heading-large">@title</h1>
-  <p>@Messages("hts.iv.were_unable_to_confirm_your_identity.p1")</p>
-  <p>@Html(Messages("hts.iv.were_unable_to_confirm_your_identity.p2"))</p>
-  <p>@Messages("hts.iv.were_unable_to_confirm_your_identity.p3")</p>
+@title = @{Messages("hts.iv.were_unable_to_confirm_your_identity.title.h1")}
 
-  <section class="subsection--wide">
-    <a href="@retryUrl" class="button" id="continue" role="button">@Messages("hts.iv.try_again_button")</a>
-  </section>
+@layout(title) {
 
+  <h1 class="govuk-heading-l">@title</h1>
+  <p class="govuk-body">@Messages("hts.iv.were_unable_to_confirm_your_identity.p1")</p>
+  <p class="govuk-body">@Html(Messages("hts.iv.were_unable_to_confirm_your_identity.p2"))</p>
+  <p class="govuk-body">@Messages("hts.iv.were_unable_to_confirm_your_identity.p3")</p>
+  @viewHelpers.govukButton(Button(
```

Comment:
we are missing an `id` attribute for this button - it used to have id = "continue"